exercises/exercise3.js

```diff
@@ -1,12 +1,92 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
-    handler: function () {
-      return 'Good Evening to You!';
+    label: 'Valley Girl Math Star, seriously',
+    prompt: 'say hi to #{name}',
+    handler: function (params) {
+      return 'Like welcome to Valley Girl Math Star demo, ' + params.name + '!';
     },
+    isReply: false,
+    isCaseSensitive: false,
+    isListening: true,
+  },
+
+  {
+    label: 'celsius to farenheit',
+    prompt: 'Enter celsius, rad conversion to farenheit #{amount as number}',
+    handler: function (params) {
+      const temperature = Math.round((params.amount * 9) / 5 + 32);
+      return 'So very ' + temperature + '!';
+    },
+    isListening: true,
```

Comment:
missing isReply?
missing isCaseSensitive?
Not required and not a bad thing, just maybe an overlooked thing?